contributing/DEVELOPERS.md

```diff
@@ -239,35 +239,6 @@ To checkout the build
     cd website && yarn build
 ```
 
-Publish on github pages __Authorized User Only__.
```

Comment:
Worth mention github action will auto public

Reply:
Yeah, I will add and netlify deployment to kepler.gl website